contracts/governance/Checkpoints.sol

```diff
@@ -1,4 +1,20 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
+//
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//
+//
+//           Trust math, not hardware.
```

Comment:
Let's find here a common motto. We can use the Threshold branding material.

Reply:
Hmmm... I was sure this is what we ended up with. Tagging @dougvk and @willthesis here.

Reply:
@cygnusv the branding material isn't for our code. do you have another suggestion? we can discuss on discord if needed

Reply:
Removed the motto, we can always add it later again.